IdentityCore/src/oauth2/token/MSIDAccessToken.h

```diff
@@ -23,15 +23,20 @@
 
 #import "MSIDBaseToken.h"
 
+@protocol MSIDAuthenticationSchemeProtocol;
+
 @interface MSIDAccessToken : MSIDBaseToken
 {
     NSString *_accessToken;
+    id<MSIDAuthenticationSchemeProtocol> _authScheme;
+    NSString *_tokenType;
 }
 
 @property (readwrite) NSDate *expiresOn;
 @property (readwrite) NSDate *extendedExpiresOn;
 @property (readwrite) NSDate *cachedAt;
 @property (readwrite) NSString *accessToken;
```

Comment:
nit: should we change accessToken to accessTokenString or something? I see several places calling like 'result.accessToken.accessToken'

Reply:
Addressed. Updated code to not have .accessToken.accessToken.